src/index.html

```diff
@@ -10,13 +10,97 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header-logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="menu-item">
+            <a
+              class="menu-link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              class="menu-link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              class="menu-link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              class="menu-link"
+              data-qa="hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              class="menu-link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              class="menu-link"
+              href="#"
```

Comment:
Same issue as in line 40. Make sure to replace `#` with the specific section id you want to link to.